src/main/java/eu/execom/labs/pomodoro/session/UserTeamDaoImpl.java

```
@@ -1,7 +1,46 @@
 package eu.execom.labs.pomodoro.session;
 
+import javax.ejb.Local;
+import javax.ejb.Stateless;
+
+import lombok.extern.slf4j.Slf4j;
+import eu.execom.labs.pomodoro.entity.Team;
+import eu.execom.labs.pomodoro.entity.User;
 import eu.execom.labs.pomodoro.entity.UserTeam;
 
-public interface UserTeamDaoImpl extends GenericDao<UserTeam, Long> {
+@Slf4j
+@Stateless
+@Local(UserTeamDao.class)
+public class UserTeamDaoImpl extends GenericDaoImpl<UserTeam, Long> implements
+        UserTeamDao {
+
+    public UserTeamDaoImpl() {
+        super(UserTeam.class);
+    }
+
+    @Override
+    public boolean inviteUser(User user) {
+        if (user == null) {
+            log.info("User is null.");
+            return false;
+        } else {
+            log.info("Adding user with {} to database...", user);
+            try {
+                // This is just a test team for now
+                UserTeam userTeam = new UserTeam();
+                userTeam.setUser(user);
+                Team team = new Team();
+                team.setName("Team1");
+                team.setTeamId(1);
+                userTeam.setTeam(team);
+                em.persist(userTeam);
+                return true;
+            } catch (Throwable e) {
```

Please don't catch *Throwable*, but a concrete exception, if any of the methods declare it, or, if you want to catch all exception, catch an *Exception*.